--- CPAN-Meta-Check-0.013/lib/CPAN/Meta/Check.pm~	2016-07-20 22:26:09.000000000 +0200
+++ CPAN-Meta-Check-0.013/lib/CPAN/Meta/Check.pm	2016-11-01 08:57:37.585737531 +0100
@@ -8,9 +8,9 @@
 our @EXPORT_OK = qw/check_requirements requirements_for verify_dependencies/;
 our %EXPORT_TAGS = (all => [ @EXPORT, @EXPORT_OK ] );
 
-use CPAN::Meta::Prereqs '2.132830';
-use CPAN::Meta::Requirements 2.121;
-use Module::Metadata 1.000023;
+use CPAN::Meta::Prereqs;
+use CPAN::Meta::Requirements;
+use Module::Metadata;
 
 sub _check_dep {
 	my ($reqs, $module, $dirs) = @_;
